docs/instant.md

```diff
@@ -637,16 +637,18 @@ This method overrides `Object.prototype.toLocaleString()` to provide a human-rea
 
 The `locales` and `options` arguments are the same as in the constructor to [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat).
 
+Since `Temporal.Instant` does not carry a time zone, the time zone used for the output will be the `timeZone` property of `options`, if present; and otherwise, the current time zone from the environment.
```

Comment:
Minor nit for a time-focused library: use "Because" not "Since".

> current time zone from the environment

I'd recommend a little more detail, e.g. current time zone from the environment, which is usually the user's or system's default time zone.  NOTE: when running Temporal code in a server environment, the server's time zone (which is usually set to `UTC`) won't match the user's time zone. So using the environment default should be avoided in that case.

Reply:
I don't think the last clarification is necessary since `toLocaleString` should only really appear in client-side code anyway, right?

Reply:
If you're building a web app and generating HTML on the server, you'll need some way to get localized data.

Reply:
You're right, it would be unfortunately if people used this on SSR-ed webapps.